src/openzaak/selectielijst/migrations/0003_move_config_to_service_model.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 2.2.10 on 2020-07-28 13:02
+
+from django.db import migrations
+
+from zgw_consumers.constants import APITypes, AuthTypes
+
+
+def move_config_to_service(apps, _):
+    ReferentieLijstConfig = apps.get_model("selectielijst", "ReferentieLijstConfig")
+    APICredential = apps.get_model("vng_api_common", "APICredential")
+    Service = apps.get_model("zgw_consumers", "Service")
+
+    config, _ = ReferentieLijstConfig.objects.get_or_create()
+
+    svc = Service.objects.filter(api_root=config.api_root).first()
+    # already exists, do nothing
+    if svc is not None:
+        return
+
+    credentials = APICredential.objects.filter(api_root=config.api_root).first()
+    Service.objects.create(
+        label=credentials.label if credentials else "VNG Selectielijst",
+        api_type=APITypes.orc,
+        api_root=config.api_root,
+        oas=f"{config.api_root}schema/openapi.yaml",
+        client_id=credentials.client_id if credentials else "",
+        secret=credentials.secret if credentials else "",
+        user_id=credentials.user_id if credentials else "",
+        user_representation=credentials.user_representation if credentials else "",
+        auth_type=AuthTypes.zgw if credentials else AuthTypes.no_auth,
```

Comment:
This is always no-auth I think? Even if a different selectielijst is used theres no way to tell if it uses ZGW auth.

Reply:
True, was trying to accommodate for any edge cases, but it makes more sense to just leave it `no_auth` and let the end-user amend if needed.